data/sql/updates/pending_db_world/DM_FactoryDoor_Fix.sql

```diff
@@ -0,0 +1 @@
+UPDATE `acore_world`.`smart_scripts` SET `action_type`=9, `comment`='Rhahk\'Zor - On Just Died - Activate Gameobject' WHERE  `entryorguid`=644 AND `source_type`=0 AND `id`=2 AND `link`=3;
```

Comment:
```suggestion
UPDATE `smart_scripts` SET `action_type`=9, `comment`='Rhahk\'Zor - On Just Died - Activate Gameobject' WHERE `entryorguid`=644 AND `source_type`=0 AND `id`=2 AND `link`=3;
```
Delete the `acore_world`. part as some people use different names for their databases :)

Reply:
Done, thanks. Will remember that for the future. :)

Reply:
The DB part is fine. The cpp part I'll leave to the others :P